digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/PointAssetOperations.scala

```diff
@@ -68,6 +68,10 @@ trait PersistedPointAsset extends PointAsset with IncomingPointAsset {
   val timeStamp: Long
   val linkSource: LinkGeomSource
   val propertyData: Seq[Property]
+
+  def getValidityDirection: Option[Int] = None
+  def getBearing: Option[Int] = None
+  def getProperty(property: String) : Option[PropertyValue] = this.propertyData.find(p => p.publicId == property).get.values.map(_.asInstanceOf[PropertyValue]).headOption
```

Comment:
varmaan vois heittää ehkä virhettä x propeties was not found.

Reply:
Pitää tutkia miten toimis liikennevalojen kanssa. Siellä kun on sekä uuden mallin mukaisia kohteita, joilla on vaikutussuunta ja vanhan mallin mukaisia, joilta puuttuu suuntatieto. Saattaa aiheuttaa ongelmia

Reply:
aa joo ne liikennemerkit jotka on vanhan lain mukaisia.
Ei silleen option heittää kyl sen NoSuchElementException, tässä vaan kerrottaisiin vähän tarkemmin mitä ei löytynyt.